packages/expect/src/spyMatchers.ts

```diff
@@ -56,7 +56,7 @@ const printReceivedArgs = (
         )
         .join(', ');
 
-const printCommon = (val: unknown) => DIM_COLOR(stringify(val));
+const printCommon = (val: unknown) => DIM_STYLE(stringify(val));
```

Comment:
Hm.. I did not suggest to change the name. Sorry, if it sounded like that. `DIM_COLOR` reads better in code: "dim color of provided string". Please keep `DIM_COLOR`. Just a detail, otherwise `noDim` option looks like a good idea.